docs/guides/testing.md

```diff
@@ -57,6 +57,11 @@ this means your application logic does not need to be changed or mocked when wri
 
 The mock provided below will enable the relevant test runner to reset the zustand stores after each test.
 
+> **Warning:** In the following examples we are only supporting the curried version of create `(create()(...))`,
+> due to that is supported on both JavaScript and Typescript. If you are using the uncurried version of
```

Comment:
```suggestion
> due to it being supported by both JavaScript and TypeScript. If you are using the uncurried version of
```

Reply:
@sewera I changed to `since it is support on` instead of `due to that is`